browser/DaisyPayments.js

```diff
@@ -47,41 +109,94 @@ export default class DaisyPayments extends ClientPayments {
     return new this.web3.eth.Contract(ERC20["abi"], invoice["tokenAddress"]);
   }
 
+  /**
+   * Takes an Web3's contract instance and wraps it into a {@link module:browser.DaisyPaymentsOnToken}.
+   * @param {external:"web3.eth.Contract"} token - Web3's contract instance from {@link module:browser~DaisyPayments#loadToken}.
+   * @returns {module:browser.DaisyPaymentsOnToken} Wrapped token.
+   */
   prepareToken(token) {
-    return new DaisyPaymentsOnToken({ manager: this.manager, token });
+    return new DaisyPaymentsOnToken({
+      manager: this.manager,
+      override: this.override,
+      withGlobals: this.withGlobals,
+      token,
+    });
   }
 }
 
-export class DaisyPaymentsOnToken {
-  constructor({ manager, token }) {
-    this.manager = manager;
+/**
+ * DaisySDK class related to operations over tokens. This should NOT be instantiated directly.
+ * Use {@link module:browser~DaisyPayments#prepareToken} to get an instance of this class.
+ *
+ * @extends module:common~DaisyPayments
+ *
+ * @example
+ *
+ * import DaisySDK from "@daisypayments/daisy-sdk/browser";
+ *
+ * const web3 = ...; // we recommend getting `web3` from [react-metamask](https://github.com/consensys/react-metamask)
+ * const daisy = await DaisySDK.initPayments({
+ *   manager: { identifier: ... }, withGlobals: { web3 },
+ * });
+ *
+ * console.log(daisy.prepareToken(daisy.loadToken(invoice)) instanceof DaisyPaymentsOnToken);
+ * // > true
+ */
+export class DaisyPaymentsOnToken extends DaisyPayments {
```

Comment:
I don't have a good idea of what the `DaisyPaymentsOnToken` class represents, but that maybe that's not a big deal since it's not instantiated directly